src/desktop/apps/auction_support/routes.coffee

```diff
@@ -35,37 +29,24 @@ registerOrRender = (sale, req, res, next) ->
                 }
               )
               res.redirect sale.registrationSuccessUrl()
-      else
-        # Render the full registration + credit card form
-        # TODO: When the bid + register flow is fully moved to reaction these routes
-        # and templates will need cleanup and templates will need deletion.
-        res.render 'registration',
-          sale: sale
-          monthRange: DateHelpers.getMonthRange()
-          yearRange: DateHelpers.getYearRange()
 
 
-@auctionRegistration = (req, res, next) ->
+@modalAuctionRegistration = (req, res, next) ->
```

Comment:
@erikdstock This was removed in the diff in the last PR which broke registration on the sale page when a user already has a credit card on file. This route is currently triggered via the modal on the sale page, so I've renamed this route handler and kept the surrounding code (thankfully not much) in.